src/tz_android.rs

```diff
@@ -0,0 +1,14 @@
+use std::env;
+use android_system_properties::AndroidSystemProperties;
+
+// From https://android.googlesource.com/platform/ndk/+/android-4.2.2_r1.2/docs/system/libc/OVERVIEW.html
+// The name of the current timezone is taken from the TZ environment variable,
+// if defined. Otherwise, the system property named 'persist.sys.timezone' is
+// checked instead.
+
+const TIMEZONE_PROP_KEY: &str = "persist.sys.timezone";
+
+pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
+    let properties = AndroidSystemProperties::new();
+    return properties.get(TIMEZONE_PROP_KEY).ok_or(crate::GetTimezoneError::OsError);
```

Comment:
In Rust, the outcome of the last statement is also the result of the function. So if the last statement it `return something;`, you can simplify it to `something`:

```rust
pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
    AndroidSystemProperties::new()
        .get(TIMEZONE_PROP_KEY)
        .ok_or(crate::GetTimezoneError::OsError)
}
```